pkg/sink/codec/debezium/codec_test.go

```diff
@@ -0,0 +1,346 @@
+// Copyright 2024 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package debezium
+
+import (
+	"math"
+	"testing"
+	"time"
+
+	"github.com/pingcap/tidb/parser/mysql"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/rowcodec"
+	"github.com/pingcap/tiflow/cdc/model"
+	"github.com/pingcap/tiflow/pkg/config"
+	"github.com/pingcap/tiflow/pkg/sink/codec/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEncodeInsert(t *testing.T) {
+	codec := &Codec{
+		config:    common.NewConfig(config.ProtocolDebezium),
+		clusterID: "test-cluster",
+		nowFunc:   func() time.Time { return time.Unix(1701326309, 0) },
+	}
+
+	e := &model.RowChangedEvent{
```

Comment:
you can take simple protocol `encoder_test.go` as an reference, it has example about how to convert DDL and DML into events.

Reply:
Thanks a lot. I will refactor the test to use SQL instead.